grammar.js

```diff
@@ -373,6 +378,39 @@ module.exports = grammar({
       seq(repeat($._annotation), '[', ']')
     )),
 
+    switch_construct: $ => seq(
+      'switch',
+      field('condition', $.parenthesized_expression),
+      field('body', $.switch_block)
+    ),
+
+    switch_block: $ => seq(
+      '{',
+      choice(repeat($.switch_line), repeat($.arrow_switch_line)),
+      '}'
+    ),
+
+    switch_line: $ => choice(
+      $.switch_label, 
+      $.statement
+    ),
+
+    switch_label: $ => choice(
+      seq('case', $.expression, ':'),
+      seq('default', ':')
+    ),
+
+    arrow_switch_line: $ => seq(
```

Comment:
I'd like avoid using the word "line" for this, since it won't necessarily be laid out on a single line, and that term doesn't seem to be used elsewhere to describe this.

What do you think about renaming this to `switch_rule` or `switch_labeled_rule`? I'm taking those terms from the language spec, which seems to use the term `SwitchLabeledRule` for this: https://docs.oracle.com/javase/specs/jls/se13/preview/switch-expressions.html#jep354-14.11.1